static/test/javascripts/spec/membership/payment-form.spec.js

```diff
@@ -61,8 +56,8 @@ define([
 
         beforeEach(function () {
             paymentFormFixtureElement = $.create(paymentFormHtml)[0];
-            paymentForm = new PaymentForm(guConfig);
-            paymentForm.init(paymentFormFixtureElement);
+            paymentForm = new PaymentForm(paymentFormFixtureElement, $(), '');
```

Comment:
wow it is interesting that the karma test still passes
I guess bonzo flattens $($(...))